### DISCUSSION TITLE
【最新文章排序】侧边栏的lastest post依据最后修改时间排序，如何改成根据发布日期来

### DISCUSSION THREAD

**Discussion by caijunqian** [tangly1024/NotionNext#2126] [Ideas]
**为什么提出这个新的特性改动**
这样当读者通过某篇文章进入到网站后，希望可以通过侧边栏及时看到最新的文章
目前侧边栏的latest post好像是依据notion中的page最后更新时间(没去阅读代码确定)排序的,
由于不是使用某个特定database中的行字段，无法控制
有时候经常点开文件不小心修改了，就导致一篇很久之前发布的文章被放到了侧边栏

**描述一下你推荐的解决方案**
使用database中的date字段，而不是page文件的修改时间(没去阅读代码确定)，或者新增一个字段等更好的方式去控制，防止误操作

**描述一下你考虑过的其它替代解决方案**
暂无

**补充说明**
暂无

**Comment by jxpeng98** [comment]
这个要看你用的是哪个主题。

如果没有更换主题的需求的话，直接在主题下边的组件文件中修改对应的内容就行了。

举个例子，如果你用的是heo主题。

你在`themes/heo/components/LatestPostsGroupMini.js`中，修改文件最后时间就可以了：
```js
<div className="text-gray-500">{post.lastEditedDay}</div>
```
为
```js
<div className="text-gray-500">{post.publishDay}</div>
```

其他主题的原理类似。
